Commit_rules.md

```diff
@@ -13,6 +13,8 @@
 * main branch는 사용하지 않습니다.  
 * nightly 브랜치에 최대한 머지합니다.  
 * nightly branch에서는 제한 없이 파생 브랜치를 만들 수 있습니다.  
+* 파생 브랜치는 name(#related_git_issue_number) 형식을 따릅니다. 
+* 커밋 메시지는 add/modified/delete_something 형식을 따릅니다
```

Comment:
Do we have reference for commit message format?
If not, I suggest we find one and decide.

Reply:
I didn't consider about commit message format for two reasons
- first, it is hard to cover every unexpected situations with rule
- second, commit message is more like "team communication" rather than "open world communication". 
So I do not want to recommend having strict format in this project. Is add/modified/delete header not enough?

Reply:
I unterstand the reasons.